.gitignore

```diff
@@ -1 +1,3 @@
 node_modules
+es
+lib
```

Comment:
but those dirs are needed for `npm publish` i guess, however are created by a `build` , its easy to run a build just to check if everything is alright without need to publish